lib/php/rendition-factory/README.md

```diff
@@ -0,0 +1,174 @@
+# Rendition Factory for video-input modules (wip)
```

Comment:
This documentation should be in the module itself.
Let's make auto-doc after

Reply:
yes